### PR TITLE
Documentation from Health-RI

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

File: content/docs/guides/LSAAI/Configure_LSAAI.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Configure KeyCloak with Azure and LSAAI
+
+* * *
+
+## Keycloak
+
+For Keycloak, I utilized a Linux app service plan within Azure. One of the primary advantages is that HTTPS is readily configured out of the box, eliminating the need for additional work with certbot. Moreover, when using an app service plan, there's no need to be concerned about Linux updates.
```
Comment (a-nayden) [inline content/docs/guides/LSAAI/Configure_LSAAI.md:7]: for documentation passive voice is preferable, e.g.: A Linux app service plan within Azure was utilised for Keycloak set up

**Review by a-nayden — CHANGES_REQUESTED** [review]
As you've mentioned - cross-links should be corrected.
for images: I guess it is reasonable to create a folder "images" for all the docs and then reference to an image in this folder so pages tree is not overloaded with not-,md files

File: content/docs/guides/LSAAI/Configure_LSAAI.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Redirect URI
+
+Both the 'Token URL' and 'Authorization URL' are derived from the IdP. When registering a service, you acquire the clientId and secret. The 'Redirect URI', which remains constant, is provided by Keycloak:  
+[https://{app_name_azure}.azurewebsites.net/auth/realms/master/broker/azuread/endpoint](https://{app_name_azure}.azurewebsites.net/auth/realms/master/broker/azuread/endpoint)
+
+Additionally, the corresponding configuration entails:
+
+* Scopes: "openid", "profile", "email"
+* Method: POST the Clientsecret
+* Sync method: import
```
Comment (a-nayden) [inline content/docs/guides/LSAAI/Configure_LSAAI.md:28]:
```suggestion
* Sync Mode: import
```

**Review by a-nayden — APPROVED** [review]
I have added a few comments on typos, other than that looks good to me - thank you

**Comment by brunopacheco1** [comment]
I will close this PR, but not merge it directly. I had to change the theme, as it was not flexible enough to allow a file-tree view, like you added. Moreover, the search/autocomplete was not working after deployed.

All your commits were preserved.

**Pull request closed without merging**